src/models/FGAuxiliary.cpp

```diff
@@ -355,15 +367,15 @@ double FGAuxiliary::GetNlf(void) const
 double FGAuxiliary::GetLongitudeRelativePosition(void) const
 {
   return in.vLocation.GetDistanceTo(FDMExec->GetIC()->GetLongitudeRadIC(),
-                                    in.vLocation.GetGeodLatitudeRad())* fttom;
+                                    in.vLocation.GetGeodLatitudeRad());
```

Comment:
It is not clear why the unit conversion from feet to meters is removed ? This method is used to compute the property `position/distance-from-start-lon-mt` which is expressed in meters.
https://github.com/JSBSim-Team/jsbsim/blob/83e3df2c49c13eae78ab7853848cdd9e02c6c5ac/src/models/FGAuxiliary.cpp#L446